exporter/loadbalancingexporter/resolver_static.go

```diff
@@ -53,7 +53,9 @@ func newStaticResolver(endpoints []string) (*staticResolver, error) {
 }
 
 func (r *staticResolver) start(ctx context.Context) error {
-	r.resolve(ctx) // right now, this can't fail
+	if _, err := r.resolve(ctx); err != nil {  // right now, this can't fail
+		log.Fatalln("error while running resolve")
+	}
 	return nil
```

Comment:
Why not just something like following?

```suggestion
	_, err := r.resolve(ctx) // right now, this can't fail
	return err
```

Reply:
done